src/rest-server/docs/swagger.yaml

```diff
@@ -1680,6 +1682,54 @@ paths:
           $ref: '#/components/responses/NoJobError'
         501:
           $ref: '#/components/responses/UnknownError'
+  /api/v1/kubernetes/nodes:
+    get:
+      tags:
+      - kubernetes
+      summary: Get kubernetes node list.
+      description: Get kubernetes node list. Need administrator permission.
+      operationId: getK8sNodes
+      security:
+      - bearerAuth: []
+      responses:
+        200:
+          description: Succeeded
+          content:
+            application/json:
+              example:
+              - Please refer to Kubernetes API doc
+              - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#list-node-v1-core
```

Comment:
use description instead?
```yaml
schema:
  type: object
  description: |
    line1
    line2
```

Reply:
I tried the description field, but I found that the description will be hided by redoc if schema's type is `object`.
![图片](https://user-images.githubusercontent.com/8802144/71337112-19166e80-2585-11ea-8ad9-317a9337c4a3.png)

And if the type is not specified, the response's description will be rendered in a collapsed div and sample response will be null.
![图片](https://user-images.githubusercontent.com/8802144/71337200-7e6a5f80-2585-11ea-89ba-0f7f3d26ef63.png)

I think it may be more straightfoward if the hints are in example field.
![图片](https://user-images.githubusercontent.com/8802144/71337237-aeb1fe00-2585-11ea-92c2-fe7d43a564d2.png)

Reply:
well, seems a redoc bug that does not render `schema.description`, SwaggerUI works fine